#!/bin/sh /usr/share/dpatch/dpatch-run
## 309-magic-add-git.dpatch by Josh Triplett <josh@freedesktop.org>
##
## DP: New magic for Git bundles (Closes: #430070).

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/revision file-4.21/magic/Magdir/revision
--- file-4.21.orig/magic/Magdir/revision	2007-04-09 16:33:07.000000000 +0000
+++ file-4.21/magic/Magdir/revision	2007-08-26 08:05:22.000000000 +0000
@@ -7,3 +7,7 @@
 # Conary changesets
 # From: Jonathan Smith <smithj@rpath.com>
 0	belong	0xea3f81bb	Conary changeset data
+
+# Type: Git bundles (git-bundle)
+# From: Josh Triplett <josh@freedesktop.org>
+0	string	#\ v2\ git\ bundle\n	Git bundle
